go.mod

```diff
@@ -2,6 +2,10 @@ module github.com/pokt-network/pocket
 
 go 1.18
 
+replace github.com/regen-network/gocuke v0.6.2 => ../gocuke
```

Comment:
Mentioned this below, your new `pokt-network/gocuke` package and same for mockdns can be used here instead?

Reply:
Yup.. this branch is outdated